templates/components/itilobject/actors/field.html.twig

```diff
@@ -269,27 +269,29 @@
           $('.popover').popover('hide');
       });
 
-       actor_select.parent().popover({
-           selector: '[data-glpi-popover-source]',
-           container: actor_select.parent(),
-           html: true,
-           sanitize: false,
-           trigger: 'hover',
-           delay: {
+      setTimeout((function() {
```

Comment:
```suggestion
      $(function () {
```
Waiting for the document to be ready should be more reliable than an arbitrary 500ms delay.